sql/telemetry_derived/clients_first_seen_v1/query.sql

```diff
@@ -0,0 +1,25 @@
+WITH today AS (
+  SELECT
+    NULL AS first_seen_date,
+    NULL AS second_seen_date,
+    * EXCEPT (submission_date)
+  FROM
+    clients_daily_v1
```

Comment:
Shouldn't this be v6?

```suggestion
    clients_daily_v6
```

Reply:
I think v6 was a poor initial choice on my part. Although clients_daily is v6, this will be the first version of clients_first_seen. Similarly, clients_last_seen is v1.

Reply:
Right but in this case we'd need to query `clients_daily_v6`? `clients_daily_v1` doesn't exist.